doc/source/standard_selections.rst

```diff
@@ -0,0 +1,95 @@
+
+The below names are drawn from the CHARMM 27, OPLS-AA, GROMOS 53A6, AMBER 03, and AMBER 99sb*-ILDN force fields.
+
++------+------+------+------+------+------+------+------+
+| ACE  | CASF | CLYS | CYS2 | HIS1 | LYN  | NGLY | NTRP |
++------+------+------+------+------+------+------+------+
+| ALA  | CASN | CME  | CYSH | HIS2 | LYS  | NHID | NTYR |
++------+------+------+------+------+------+------+------+
+| ALAD | CASP | CMET | CYX  | HISA | LYSH | NHIE | NVAL |
++------+------+------+------+------+------+------+------+
+| ARG  | CCYS | CPHE | DAB  | HISB | MET  | NHIP | ORN  |
++------+------+------+------+------+------+------+------+
+| ARGN | CCYX | CPRO | GLH  | HISD | MSE  | NILE | PGLU |
++------+------+------+------+------+------+------+------+
+| ASF  | CGLN | CSER | GLN  | HISE | NALA | NLEU | PHE  |
++------+------+------+------+------+------+------+------+
+| ASH  | CGLU | CTHR | GLU  | HISH | NARG | NLYS | PRO  |
++------+------+------+------+------+------+------+------+
+| ASN  | CGLY | CTRP | GLUH | HSD  | NASN | NME  | QLN  |
++------+------+------+------+------+------+------+------+
+| ASN1 | CHID | CTYR | GLY  | HSE  | NASP | NMET | SER  |
++------+------+------+------+------+------+------+------+
+| ASP  | CHIE | CVAL | HID  | HSP  | NCYS | NPHE | THR  |
++------+------+------+------+------+------+------+------+
+| ASPH | CHIP | CYM  | HIE  | HYP  | NCYX | NPRO | TRP  |
++------+------+------+------+------+------+------+------+
+| CALA | CILE | CYS  | HIP  | ILE  | NGLN | NSER | TYR  |
++------+------+------+------+------+------+------+------+
+| CARG | CLEU | CYS1 | HIS  | LEU  | NGLU | NTHR | VAL  |
++------+------+------+------+------+------+------+------+
+
+----------------
+Protein backbone
+----------------
+
+Protein backbone atoms in MDAnalysis belong to a recognised protein residue and have the atom names CA, C, O, N. 
+
+.. _nucleic-selection:
+
+Nucleic acids
+=============
+
+The residue names listed here are accessible via the "nucleic" keyword in the :ref:`selections`. 
+
+The below names are drawn from largely from the CHARMM force field.
+
++-----+----+----+---+-----+-----+-----+-----+
+| ADE | DA | RU | T | DA5 | DA3 | RA5 | RA3 |
++-----+----+----+---+-----+-----+-----+-----+
+| URA | DC | RG | U | DC5 | DC3 | RU5 | RU3 |
++-----+----+----+---+-----+-----+-----+-----+
+| CYT | DG | RC | C | DG5 | DG3 | RG5 | RG3 |
++-----+----+----+---+-----+-----+-----+-----+
+| GUA | DT | A  | G | DT5 | DT3 | RC5 | RC3 |
++-----+----+----+---+-----+-----+-----+-----+
+| THY | RA |    |   |     |     |     |     |
++-----+----+----+---+-----+-----+-----+-----+
+
+----------------
+Nucleic backbone
+----------------
+
+Nucleic backbone atoms in MDAnalysis belong to a recognised nucleic acid residue and have the atom names P, C5', C3', O3', O5'.
+
+.. _nucleobase-selection:
+
+-----------
+Nucleobases
+-----------
+
+Nucleobase atoms from nucleic acid residues are recognised based on their names in CHARMM.
```

Comment:
Is there a way to generate this table programmatically from the MDAnalysis version that the UserGuide pertains to?

Although this is not likely to change soon, when it does I guarantee that someone will forget to update the UserGuide.

I think @jbarnoud has written code to, for instance, dynamically create the AUTHORS list in the docs.